source/GlassView.Export/GlassView.cs

```diff
@@ -13,13 +16,20 @@ namespace Atmoos.GlassView.Export;
 public static class GlassView
 {
     private const String glassViewSection = nameof(GlassView);
+    private static readonly DirectoryName glassViewDirectory = new(glassViewSection);
+    private static readonly DirectoryName artifactDirectory = new("BenchmarkDotNet.Artifacts");
     public static IGlassView Default() => Default(ConsoleLogger.Default);
-    public static IGlassView Default(ILogger logger)
-        => new SummaryExporter(new DirectoryExport(GlassViewArtifactsDirectory(), SerializationOptions()), logger);
+    public static IGlassView Default(ILogger logger) => Default<Current>(logger);
```

Comment:
What is `Current`? I don't see it defined in this class...

Reply:
The full name of "Current" is: `Atmoos.World.IO.FileSystem.Current`
The same is also true for time: `Atmoos.World.Time.Current`